### ISSUE TITLE
LoRA support?

### ISSUE THREAD

**Issue by psych0v0yager** [karpathy/llama2.c#11]
Are there any plans to offer LoRA support in the future? Currently I have been using this library (https://github.com/cccntu/minLoRA) with nanoGPT 

By the way, huge fan of your videos. I loved the way you coded language models live as well as providing priceless intuition on all the core concepts. I would love to see a brief video on this repository as well as all the latest innovations in the llm space (alibi, rotary embeddings, flash attention, lora, quantization, etc)

**Comment by karpathy** [comment]
Yep, definitely. LoRA would be near trivial to add to nanoGPT I just didn't want to clutter it too much. As for llama2.c LoRA is a bit less relevant or useful, unless we eventually get very large models working in low precision.
